### ISSUE TITLE
Has anything changed of hyper-pararmeter on the Affectnet dataset?

### ISSUE THREAD

**Issue by elk-april** [zengqunzhao/MA-Net#2]
Hello, it is a nice work. 
But I can't reproduce the results of paper in Affectnet using the setting of this repo.
Could you tell me the mainly changed setting?
Ps . Imbalanced Sample is using。。。

**Comment by zengqunzhao** [comment]
> Hello, it is a nice work.
> But I can't reproduce the results of paper in Affectnet using the setting of this repo.
> Could you tell me the mainly changed setting?
> Ps . Imbalanced Sample is using。。。

Thanks for your attetnion.
Did you reproduce it on RAF-DB?

The hyper-parameters for AffectNet are as follow:
epoch: 20, batch_size: 256, lr: 0.01
scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=5, gamma=0.1)

**Comment by elk-april** [comment]
Awesome， I will try it.
